andela/blackninjas-frontend
# #171351272 Implement admin assign users managers

#### What does this PR do?
Implementation of admin should be able to assign users their managers
#### Description of Task to be completed?
- Implement a table of users and their managers
- Implement a list of all managers in a dropdown
- Implement update of multiple records at once
#### How should this be manually tested?
- clone this repo
- make sure you have all installed packages
- run the server by typing `npm run start-dev` and make sure you have the privileges of an admin and you should see a table of users and their managers change their managers and hit send
#### Any background context you want to provide?
- Assigning users to their managers will be in the admin's privilege
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/51836855/75181542-24ce0f00-5747-11ea-942b-bedc80b81a61.png)
![image](https://user-images.githubusercontent.com/51836855/76323250-22de8100-62ed-11ea-8dcd-44ff02f92377.png)
![image](https://user-images.githubusercontent.com/51836855/76323414-5c16f100-62ed-11ea-82b0-6967a8e0a30a.png)


#### Questions:
